torch/fx/__init__.py

```diff
@@ -90,4 +90,4 @@ def forward(self, x):
 from .symbolic_trace import symbolic_trace, Tracer
 from .graph import Graph
 from .node import Node, map_arg
-from .proxy import Proxy
+from .proxy import Proxy, len
```

Comment:
It doesn't seem to be causing typechecking issues in this PR, but do you mind adding an import for `len` to `torch/fx/__init__.pyi` as well? There are some situations where importing `torch.fx.foo` where foo is imported in the `__init__.py` but not in the `__init__.pyi` causes issues